src/test/kotlin/lotto/model/LottoMachineTest.kt

```diff
@@ -0,0 +1,53 @@
+package lotto.model
+
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class LottoMachineTest {
+    @ValueSource(ints = [-1, 0])
+    @ParameterizedTest
+    fun `입력한 금액은 0원 이상만 가능하다`(amount: Int) {
+        assertThatThrownBy {
+            LottoMachine(0)
+        }.hasMessageContaining("0원 이상의 금액")
+    }
+
+    @Test
+    fun `입력한 금액이 1,000으로 나누어지지 않으면 실패한다`() {
+        assertThrows<IllegalArgumentException> { LottoMachine(1001) }
+    }
+
+    @Test
+    fun `구입 금액이 5000원이면 로또를 5개 반환한다`() {
+        val amount = 5000
+        val lottoMachine = LottoMachine(amount)
+        val lottoQuantity = lottoMachine.getLottoQuantity()
+
+        Assertions.assertEquals(5, lottoQuantity)
```

Comment:
nit: 아래처럼 간단하게 작성할 수도 있어요.
```suggestion
        import org.junit.jupiter.api.Assertions.assertEquals
        assertEquals(5, lottoQuantity)
```
모든 테스트를 확인해주세요 🤠

Reply:
아앗 이런! 임포트가 제대로 되어 있지 않은 곳이 많네요 ㅠ Assertions 내부의 함수가 뭐가 있을까! 를 찾다보다 그랬나봐요 🥲